zone/raids.cpp

```diff
@@ -1486,70 +1605,84 @@ void Raid::SaveRaidMOTD()
 
 bool Raid::LearnMembers()
 {
-	memset(members, 0, (sizeof(RaidMember)*MAX_RAID_MEMBERS));
+	memset(members, 0, (sizeof(RaidMember) * MAX_RAID_MEMBERS));
 
-	std::string query = StringFormat("SELECT name, groupid, _class, level, "
-                                    "isgroupleader, israidleader, islooter "
-                                    "FROM raid_members WHERE raidid = %lu",
-                                    (unsigned long)GetID());
-    auto results = database.QueryDatabase(query);
-    if (!results.Success())
-        return false;
-
-	if(results.RowCount() == 0) {
-		LogError("Error getting raid members for raid [{}]: [{}]", (unsigned long)GetID(), results.ErrorMessage().c_str());
-        disbandCheck = true;
-        return false;
-    }
-
-    int index = 0;
-    for(auto row = results.begin(); row != results.end(); ++row) {
-        if(!row[0])
-            continue;
-
-        members[index].member = nullptr;
-        strn0cpy(members[index].membername, row[0], 64);
-		uint32 groupNum = Strings::ToUnsignedInt(row[1]);
-        if(groupNum > 11)
-            members[index].GroupNumber = 0xFFFFFFFF;
-        else
-            members[index].GroupNumber = groupNum;
-
-        members[index]._class = Strings::ToInt(row[2]);
-        members[index].level = Strings::ToInt(row[3]);
-        members[index].IsGroupLeader = Strings::ToInt(row[4]);
-        members[index].IsRaidLeader = Strings::ToInt(row[5]);
-        members[index].IsLooter = Strings::ToInt(row[6]);
-        ++index;
-    }
+	const auto query = fmt::format(
+		"SELECT name, groupid, _class, level, "
+		"isgroupleader, israidleader, islooter, bot_id "
+		"FROM raid_members WHERE raidid = {} ORDER BY groupid",
+		GetID()
+	);
+
+	auto results = database.QueryDatabase(query);
+	if (!results.Success()) {
+		return false;
+	}
+
+	if (!results.RowCount()) {
+		LogError("Error getting raid members for raid [{}]: [{}]", GetID(), results.ErrorMessage());
```

Comment:
Is this truly an error? Is this something actionable that the server operator should be aware of? If not we should remove it